src/shogun/features/Features.cpp

```diff
@@ -272,17 +272,6 @@ CFeatures* CFeatures::view(const SGVector<index_t>& subset)
 	return feats_view;
 }
 
-CFeatures* CFeatures::view(const std::vector<index_t>& subset)
```

Comment:
can you explain ?
We need to use the base class interface in things like xvalidation ..

Reply:
ah sorry nevermind I didnt see the arugment was the std

Reply:
instead: yes ! :)